.eslintrc

```diff
@@ -15,13 +14,14 @@
     "curly": [2, "all"],
     "dot-notation": [2, {"allowKeywords": true}],
     "eqeqeq": [2, "allow-null"],
-    "global-strict": [2, "always"],
+    "global-strict": 0,
```

Comment:
It's not common js so we don't want a global strict rule. Also I just wanted to quieten things down because this is just the example.